libs/net/services/Actions/Managers/IngestActionManager`.cs

```diff
@@ -104,6 +104,20 @@ private async Task<IngestModel> UpdateIngestAsync(int failedAttempts = 0)
         return await this.Api.UpdateIngestAsync(Ingest, headers) ?? Ingest;
     }
 
+    /// <summary>
+    /// Update Ingest config at runtime.
+    /// </summary>
+    /// <param name="propName"></param>
+    /// <param name="propValue"></param>
+    /// <returns></returns>
+    public override async Task UpdateIngestConfigAsync(string propName, object propValue)
+    {
+        this.Ingest.Configuration[propName] = propValue;
+        var headers = new HttpRequestMessage().Headers;
+        headers.Add("User-Agent", GetType().FullName);
+        this.Ingest = await this.Api.UpdateIngestAsync(Ingest, headers) ?? Ingest;
```

Comment:
Don't need to handle presently, but a concurrency error will result in a failed import.